src/extensiveFormulation.jl

```diff
@@ -95,9 +95,10 @@ function extensive_formulation(model,
     solved = (status == :Optimal)
 
     if solved
-        return getObjectiveValue(mod), status
+        firstControl = collect(values(getValue(u)))[1:DIM_CONTROL*laws[1].supportSize]
+        return getObjectiveValue(mod), firstControl, status
     else
-        return -1., status
+        return -1., -1., status
```

Comment:
I think we should at least throw a warning, maybe an error. -1 could be a sensible solution.